.lando.yml

```diff
@@ -72,6 +72,10 @@ tooling:
     dir: /app/drupal/oauth
     user: www-data
 
+  redis-cli:
+    cmd:
+      - cache: redis-cli -h cache -p 6379 -a mypassword
+
```

Comment:
This adds a utility command to lando to debug redis.

In addition to this we can also use Redis Insight